static/js/dark-init.js

```diff
@@ -14,6 +14,13 @@
     // ignore errors e.g. when media query matching is not supported
   }
 
+  // NOTE: dartdoc currently does not support externally supported color scheme,
+  //       and we disable that part for the affected URLs.
+  // TODO: remove this part when dartdoc supports the above detection too
```

Comment:
Can we create an issue for dartdoc for this TODO?

Reply:
https://github.com/dart-lang/dartdoc/issues/4069